Kahi_impactu_postcalculations/kahi_impactu_postcalculations/Kahi_impactu_postcalculations.py

```diff
@@ -114,26 +123,41 @@ def run(self):
                         self.author_count
                     ) for oaid in authors_ids)
 
-        # Getting the top words for each institution
-        print("INFO: Creating top words for institutions")
+        # Getting the top words for institution - step 1
+        if self.verbose > 1:
+            print("INFO: Creating top words for institutions - step 1")
```

Comment:
step 1 do no mean nothing for the user, lets say ```print("INFO: Creating top words for institutions")```

and in the other step lets put
```print("INFO: Creating top words for faculty, department and group ")```